src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -703,7 +703,6 @@ void BroadcastUpdate(string action)
 	{
 		if (Build.VERSION.SdkInt >= BuildVersionCodes.Tiramisu)
 		{
-			Logger.LogError("{LocalBroadcastManager} not supported on Android 13 and above.", typeof(LocalBroadcastManager));
```

Comment:
Is it worth a comment here for future developers to indicate why we don't need to do anything on later versions?